nodejs-jquery/nodejs-jquery.spec

```diff
@@ -0,0 +1,57 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name jquery
+
+Name: %{?scl_prefix}nodejs-%{npm_name}
+Version: 1.11.3
+Release: 1%{?dist}
+Summary: JavaScript library for DOM operations
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/jquery/jquery.git
+Source0: http://registry.npmjs.org/jquery/-/jquery-1.11.3.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires: nodejs(engine)
+BuildRequires: nodejs-devel
+BuildRequires: nodejs-packaging
+BuildArch: noarch
+ExclusiveArch: %{nodejs_arches} noarch
+Provides: %{?scl_prefix}npm(%{npm_name}) = %{version}
+
+%description
+JavaScript library for DOM operations
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+This package contains documentation for nodejs-%{npm_name}.
+
+%prep
+%setup -q -n package
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}%{nodejs_sitelib}/jquery
+cp -pfr dist src  *.json *.md *.js *.txt node_modules %{buildroot}%{nodejs_sitelib}/jquery > /dev/null 2>&1 || :
+
+%build
+
+%check
+%nodejs_symlink_deps --check
+
+%files
+%defattr(-,foreman,foreman,-)
```

Comment:
These shouldn't be owned by `foreman` as the package doesn't depend on foreman for the account, nor does foreman need write access. Perhaps leave this out?